src/widgets/course-main/utils/get-course-status.ts

```diff
@@ -1,7 +1,10 @@
 import dayjs from 'dayjs';
+import isBetween from 'dayjs/plugin/isBetween';
 import { hasDayInDate } from './has-day';
 import type { CourseStatus } from '@/entities/course';
 
+dayjs.extend(isBetween);
```

Comment:
Maybe we should do something like this https://github.com/rolling-scopes/site/commit/4456ce8b8777ba1f7857d904177a485547aa1a36#diff-e9cfeca12ef7bd0421bf17b6b8b868825d8be75a249bf84b76d5197cbc6771f1R5

Reply:
done